packages/flutter_tools/test/integration.shard/android_gradle_print_app_link_domains_test.dart

```diff
@@ -136,6 +135,13 @@ void main() {
     tryToDelete(tempDir);
   });
 
+  void testDeeplink(dynamic deeplink, String scheme, String host, String path) {
```

Comment:
Can you add a test for an app with no links and verifies the empty format. Does it error? are the values null? blank file etc?